owncloudData/src/test/java/com/owncloud/android/data/sharing/shares/repository/OCShareRepositoryTest.kt

```diff
@@ -19,497 +19,281 @@
-    private val ocShareRepository: OCShareRepository =
-        OCShareRepository(localShareDataSource, remoteShareDataSource)
-
-    private val shares = arrayListOf(
-        OC_PUBLIC_SHARE,
-        OC_PRIVATE_SHARE,
-        OC_PRIVATE_SHARE,
-        OC_PUBLIC_SHARE
-    )
 
-    private val filePath = "/Images"
-    private val accountName = "admin@server"
-
-    @Test
-    fun refreshSharesFromNetworkOk() {
-        every { remoteShareDataSource.getShares(any(), any(), any(), any()) } returns shares
+    private val localShareDataSource = mockk<LocalShareDataSource>(relaxUnitFun =  true)
+    private val remoteShareDataSource = mockk<RemoteShareDataSource>(relaxUnitFun = true)
+    private val ocShareRepository = OCShareRepository(localShareDataSource, remoteShareDataSource)
 
-        ocShareRepository.refreshSharesFromNetwork(filePath, accountName)
-
-        verify(exactly = 1) {
-            remoteShareDataSource.getShares(
-                remoteFilePath = filePath,
-                reshares = true,
-                subfiles = false,
-                accountName = accountName
-            )
-        }
-
-        verify(exactly = 1) {
-            localShareDataSource.replaceShares(shares)
-        }
-    }
+    private val listOfShares = listOf(OC_PRIVATE_SHARE, OC_PUBLIC_SHARE)
+    private val filePath = "/Images"
+    private val password = "password"
+    private val permissions = 1
+    private val expiration = RemoteShare.INIT_EXPIRATION_DATE_IN_MILLIS
 
     @Test
-    fun refreshSharesFromNetworkEmptyShares() {
-        every { remoteShareDataSource.getShares(any(), any(), any(), any()) } returns listOf()
-
-        ocShareRepository.refreshSharesFromNetwork(filePath, accountName)
-
-        verify(exactly = 1) {
-            remoteShareDataSource.getShares(
+    fun `insertPrivateShare inserts a private OCShare correctly`() {
+        every {
+            remoteShareDataSource.insert(
                 remoteFilePath = filePath,
-                reshares = true,
-                subfiles = false,
-                accountName = accountName
+                shareType = OC_PRIVATE_SHARE.shareType,
+                shareWith = OC_SHAREE.shareWith,
+                permissions = permissions,
+                name = "",
+                password = "",
+                expirationDate = expiration,
+                accountName = OC_ACCOUNT_NAME
             )
-        }
-
-        verify(exactly = 1) {
-            localShareDataSource.deleteSharesForFile(filePath, accountName)
-        }
-    }
+        } returns OC_PRIVATE_SHARE
 
-    @Test(expected = NoConnectionWithServerException::class)
-    fun refreshSharesFromNetworkNoConnection() {
-        every { remoteShareDataSource.getShares(any(), any(), any(), any()) } throws NoConnectionWithServerException()
+        every {
+            localShareDataSource.insert(OC_PRIVATE_SHARE)
+        } returns 1
```

Comment:
Add a comment like
``// The result of this method is not used, so it can be anything``
above each of these `every`s mocking something that won't be used

Reply:
I've added that comment to all `every` block that returns something that is not used 🚀